live/Artillery/0.0.8.ini

```diff
@@ -0,0 +1,1238 @@
+bridge_angle = 0
+bridge_flow_ratio = 1
+bridge_speed = 30
+brim_separation = 0
+brim_type = outer_only
+brim_width = 0
+compatible_printers =
+compatible_printers_condition =
+complete_objects = 0
+dont_support_bridges = 1
+draft_shield = disabled
+elefant_foot_compensation = 0
+external_perimeters_first = 0
+extra_perimeters = 1
+extra_perimeters_on_overhangs = 0
+extruder_clearance_height = 20
+extruder_clearance_radius = 20
+extrusion_width = 0.45
+fill_angle = 45
+fill_pattern = alignedrectilinear
+first_layer_acceleration_over_raft = 0
+first_layer_height = 0.25
+first_layer_speed = 30
+first_layer_speed_over_raft = 30
+fuzzy_skin = none
+fuzzy_skin_point_dist = 0.8
+fuzzy_skin_thickness = 0.3
+gap_fill_enabled = 1
+gap_fill_speed = 20
+gcode_comments = 0
+gcode_label_objects = 0
+gcode_resolution = 0.0125
+gcode_substitutions =
+infill_acceleration = 0
+infill_anchor = 600%
+infill_anchor_max = 50
+infill_every_layers = 1
+infill_extruder = 1
+infill_extrusion_width = 0.45
+infill_first = 0
+inherits =
+interface_shells = 0
+ironing = 0
+ironing_flowrate = 15%
+ironing_spacing = 0.1
+ironing_speed = 15
+ironing_type = top
+layer_height = 0.2
+max_volumetric_extrusion_rate_slope_negative = 0
+max_volumetric_extrusion_rate_slope_positive = 0
+max_volumetric_speed = 0
+min_bead_width = 85%
+min_feature_size = 25%
+min_skirt_length = 0
+mmu_segmented_region_interlocking_depth = 0
+mmu_segmented_region_max_width = 0
+notes =
+only_retract_when_crossing_perimeters = 0
+ooze_prevention = 0
+output_filename_format = {input_filename_base}_{layer_height}mm_{filament_type[0]}_{printer_model}_{print_time}.gcode
+perimeter_acceleration = 1500
+perimeter_speed = 120
+seam_position = rear
+skirts = 0
+skirt_distance = 0
+small_perimeter_speed = 25
+solid_infill_extrusion_width = 0.45
+support_material_extrusion_width = 0.35
+top_infill_extrusion_width = 0.4
+travel_acceleration = 3000
+travel_speed = 250
+external_perimeter_extrusion_width = 0.4
+
+[print:*common_x3*]
+inherits = *common_x3_x4*
+compatible_printers_condition = printer_model=~/(X3).*/ and nozzle_diameter[0]==0.4
+avoid_crossing_perimeters = 0
+bridge_acceleration = 1000
+default_acceleration = 3000
+enable_dynamic_overhang_speeds = 1
+external_perimeter_acceleration = 1500
+external_perimeter_extrusion_width = 0.45
+fill_density = 5%
+first_layer_acceleration = 0
+first_layer_extrusion_width = 0.42
+infill_overlap = 25%
+max_print_speed = 400
+solid_infill_speed = 150
+top_solid_infill_speed = 150
+
+[print:X3 Default (0.20mm) @Artillery]
+inherits = *0.20mm*
+external_perimeter_speed = 35
+fill_density = 10%
+fill_pattern = grid
+first_layer_height = 0.25
+gap_fill_speed = 45
+infill_speed = 150
+infill_only_where_needed = 1
+perimeter_speed = 80
+perimeters = 2
+solid_infill_speed = 150
+top_solid_infill_speed = 50
+travel_speed = 250
+compatible_printers_condition = printer_model=~/(X3).*/ and nozzle_diameter[0]==0.4
+
+[print:ArtillerySlicer X3 ABS/PETG/PLA_Silk @Artillery]
```

Comment:
This is quite strange print preset naming. Is it really a good idea to name print profiles after other slicer? I understand the settings were inspired by values in different slicer, but it still feels odd.

Reply:
Also, you cannot use some characters in the preset name, like `/`.

Reply:
![characters](https://github.com/prusa3d/PrusaSlicer-settings/assets/36745189/707b5a7a-668b-4ee7-ac50-9ed0212b0bb3)

Reply:
> This is quite strange print preset naming. Is it really a good idea to name print profiles after other slicer? I understand the settings were inspired by values in different slicer, but it still feels odd.

This is only because the profiles were not inspired, but completely copied (and only slightly optimised to take up about 1k lines of code instead of over 6k)